apps/examples/e2e/tests/fixtures/menus/stylepanel.ts

```diff
@@ -0,0 +1,18 @@
+import { Page } from '@playwright/test'
+
+export class StylePanel {
+	readonly stylesArray: string[]
+
+	constructor(public readonly page: Page) {
+		this.page = page
+		this.stylesArray = [
```

Comment:
It would be best if we could get this straight from code. Then the code and the tests could not diverge. But it doesn't seem like we keep this organized in some sort of an array. So it's going to be hard to know that we have to update our tests if we add an additional style option to the styles panel. But I guess it's fine for now.